pkg/shared/families/malware/yara/config/config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
```

Comment:
Why we need to put the `config.go` under its own subdirectory? I'd put it under the `yara` directory instead and avoid subdirectories for packages with single files. `Config` could live in the `yara` pacakge so importing it would be more straightforward instead of using aliases.

I prefer this:

```go
import "github.com/openclarity/vmclarity/pkg/shared/families/malware/yara/"

...

type ScannersConfig struct {
        ...
	Yara yara.Config `yaml:"yara" mapstructure:"yara"`
}

```

Instead of this: https://github.com/openclarity/vmclarity/pull/638/files#diff-6efd9504dcd8edbb0bd19fbc5274207a9216b1bb81d5b7ad9f7e87409f3ce152R18-R21

Reply:
It causes some import cycle because we are using them in the common.Config that is used by both yara and clam.
We need to do some refactoring to achieve this.